### DISCUSSION TITLE
能搞个更新日志吗

### DISCUSSION THREAD

**Discussion by nikissXI** [linweiyuan/go-chatgpt-api#117] [Ideas]
镜像有没有更新，更新了什么啥都不知道qwq

**Comment by linweiyuan** [comment]
这些就是更新日志，绿色打勾就是有新镜像推送成功

最新版本总是能用，但不保证兼容旧版，如果你想确保一直运行的是最新版，跑个 `watchtower` 定期检测新镜像，也就占一点内存

![2023-05-09_17-07](https://user-images.githubusercontent.com/49076004/237049351-91ae7114-8f95-4bfb-be30-1a9851799ef8.png)

**Reply by nikissXI** [reply]
好的！
